src/borg/archiver/_common.py

```diff
@@ -249,9 +249,37 @@ def wrapper(self, args, repository, manifest, **kwargs):
 class Highlander(argparse.Action):
     """make sure some option is only given once"""
 
+    def __init__(
+        self,
+        option_strings,
+        dest,
+        nargs=None,
+        const=None,
+        default=None,
+        type=None,
+        choices=None,
+        required=False,
+        help=None,
+        metavar=None,
```

Comment:
is there a need to give these defaults or could one also just use `*args, **kwargs` here?